source/releasenotes/2024-02-27-wp-cli-changes.md

```diff
@@ -0,0 +1,22 @@
+---
+title: WP-CLI Pantheon wrapper changes
+published_date: "2024-02-27"
+categories: [wordpress,action-required]
+---
+
+We've recently modified the Pantheon wrapper around the WP-CLI executable to solve a number of interrelated issues that customers may have previously been experiencing and/or using workarounds to solve.
+
+<h3>The Problem</h3>
+
+* Our WP-CLI wrapper included a hard-coded `--url` flag for all `wp` commands run on the platform (via Terminus, e.g. `terminus remote:wp` or `terminus wp` or internal workflows that relied on WP-CLI like flushing cache or viewing the Status page in the dashboard).
+* For WordPress multisites, the hard-coded `--url` flag pointed to the Pantheon platform domain (`*.pantheonsite.io`) instead of the actual domain of the site resulting in errors and the command failing unless executed manually (via Terminus with the correct `--url` flag attached).
```

Comment:
Worth clarifying this is/was primary domain set in the dashboard, or not worth the distinction?